main.py

```diff
@@ -0,0 +1,8 @@
+from fastapi import FastAPI
+from city.router import router as city_router
+from temperature.router import router as temperature_router
```

Comment:
Ensure that the `temperature.router` module is correctly defined and that it exports a `router` object. This is necessary for the application to handle temperature-related routes.